azurerm/provider.go

```diff
@@ -85,6 +85,7 @@ func Provider() terraform.ResourceProvider {
 			"azurerm_client_config":                         dataSourceArmClientConfig(),
 			"azurerm_cosmosdb_account":                      dataSourceArmCosmosDBAccount(),
 			"azurerm_dns_zone":                              dataSourceArmDnsZone(),
+			"azurerm_data_lake_store":                       dataSourceArmDataLakeStoreAccount(),
```

Comment:
minor: alphabetically this should be above `azurerm_dns_zone`

Reply:
moved, in next commit